client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+            Elementul
+            {' '}
+            <strong className="formatted">{'<a>'}</strong>
+            {' '}
+            denumit și
+            {' '}
+            <i>ancoră</i>
+            {' '}
+            sau
+            {' '}
+            <i>link</i>
+            {' '}
+            este unul din cele mai uzuale elemente
+            folosite pentru construirea paginilor web. Dacă vrem să ne mutam la
+            o altă pagină sau la o altă zonă în cadrul aceleiași pagini, atunci
+            vom folosi acest element. Modalitatea prin care adăugăm un link
+            într-o pagină este prin folosirea atributului
+            {' '}
+            <strong className="formatted">href</strong>
+            {' '}
+            care indică URL-ul
+            link-ului.
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="exemple-unde-folosim-tagul-a">
+            Exemple unde folosim tag-ul a
+          </LessonHeading>
+          <LessonHeading as="h4" id="cele-mai-uzuale">
+            Cele mai uzuale
+          </LessonHeading>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="https://frontend.ro/"> Frontend.ro </a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/link-to-a-page.png"
+            alt="Cel mai uzual mod de a folosi tag-ul a - URL-uri absolute"
+            demo="/demo/html/url-absolut"
+          />
+          <p>
+            În cazul acesta, spunem că este un link către un URL
+            {' '}
+            <strong>absolut</strong>
+            {' '}
+            - adică un URL care conține întreaga adresă
+            a unei pagini. Acest tip de URL este deseori întâlnit atunci când un
+            website face legătura către un alt website.
+          </p>
+          <p>
+            Putem avea și URL-uri
+            {' '}
+            <strong>relative</strong>
+            , de obicei folosite
+            la realizarea meniurilor paginilor web:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/team.html">Team</a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/menu-github.png"
+            alt="URL-uri relative"
+            demo="/demo/html/url-relativ"
+          />
+          <p>
+            Foarte utile sunt și link-urile care duc către o secțiune ale
+            aceleiași pagini sau așa numitele
+            {' '}
+            <strong className="formatted">jump links</strong>
+            .
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="#Section-further-down"> 
+  Sari la secțiunea de mai jos 
+</a> 
+<h2 id="Section-further-down">Section further down</h2>
+            `}
+          />
+          <LessonFigure
```

Comment:
Pretty useless demo when compared to the video below. Let's remove and keep the video. 

When clicking the video it should navigate you to the demo page (pattern already exists in other articles) which should actually have stuff you jump to. Right now the demo page doesn't do anything, you click the link and nothing happens because there's no scroll. People might think it's broken.